### PR TITLE
[ADXT-591] Load balance fakeintake service between several ECS clusters

### PR DESCRIPTION
**Pull request by KevinFairise2**
What does this PR do?
---------------------

Load balances fakeintake services between several ECS clusters.
This should help to reduce the load on each ECS cluster and reduce the issues we noticed we have a high number of fakeintake created in a short time

Needs: https://github.com/DataDog/cloud-inventory/pull/22922

Which scenarios this will impact?
-------------------

Motivation
----------

Additional Notes
----------------

### CODE REVIEW

**Review by pducolin — APPROVED** [review]

File: resources/aws/environmentDefaults.go [hunk 1]
```diff
@@ -101,14 +101,14 @@ func sandboxDefault() environmentDefault {
 			defaultInstanceType:            "t3.medium",
 			defaultInstanceProfileName:     "ec2InstanceRole",
 			defaultARMInstanceType:         "t4g.medium",
-			defaultInstanceStorageSize:     200,
+			defaultInstanceStorageSize:     50,
```
Comment (pducolin) [inline resources/aws/environmentDefaults.go:104]: **❓ question**
Should this go in a separate PR?

Comment (KevinFairise2) [reply resources/aws/environmentDefaults.go:104]: Yes it should, I have a local commit that I forgot to push removing this change, thanks!
